test/suites/smoke-test-common/test-consistency-services-payment.ts

```diff
@@ -0,0 +1,69 @@
+import "@tanssi/api-augment";
+import { beforeAll, describeSuite, expect } from "@moonwall/cli";
+import { ApiPromise } from "@polkadot/api";
+import { paraIdTank } from "util/payment";
+
+describeSuite({
+    id: "S09",
+    title: "Check services payment consistency",
+    foundationMethods: "read_only",
+    testCases: ({ it, context }) => {
+        let api: ApiPromise;
+        let runtimeVersion;
+        const costPerSession = 100_000_000n;
+        const costPerBlock = 1_000_000n;
+        let blocksPerSession;
+
+        beforeAll(async () => {
+            api = context.polkadotJs();
+            runtimeVersion = api.runtimeVersion.specVersion.toNumber();
+            const chain = api.consts.system.version.specName.toString();
+            blocksPerSession = chain == "Dancebox" ? 300n : 5n;
+        });
+
+        it({
+            id: "C01",
+            title: "All scheduled parachains should be able to pay for at least 2 sessions",
+            test: async function () {
+                if (runtimeVersion < 500) {
+                    return;
+                }
+                const existentialDeposit = await api.consts.balances.existentialDeposit.toBigInt();
+
+                // If they have collators scheduled, they should have at least enough money to pay
+                const pending = await api.query.collatorAssignment.pendingCollatorContainerChain();
```

Comment:
Here if this is None it means it has not changed, you can use `api.query.collatorAssignment.collatorContainerChain()`.